### ISSUE TITLE
Accessibility problems with menu bar and lists

### ISSUE THREAD

**Issue by afr-ux** [home-assistant/frontend#25932]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [x] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Hello,
in the latest public versions of homeassistant there are serious accessibility problems for screen reader users, in addition to the existing ones. I will comment some of them:

- Since they changed the side panel, I think in 2025.3, the accessibility on PC has dropped a lot.

- With the new version of HomeAssistant and JAWS/NVDA, now I see a space in the main menu between each of the items and if I hit the toggle button on the side panel, in addition to not specifying whether it is expanded or collapsed, I see a list, which in my case is 18 items, but I see it empty.

- The drop-down lists of the different panels are broken, there are options that can not be selected, for example in JAWS you have to disable the virtual cursor, select the option and return to the virtual view of the page, in add-ons, for example, you can not or it costs a lot to select the add-ons you have installed, in other menus the same thing happens, all this added to the difficulties that already existed in previous versions to create automations, tools for developers, etc., and also to create panels, etc..  In short, a small disaster. to move from one panel to another you have to do a lot of work, the same happens in configuration, system, etc..

- For the side panel, which used to show a button that you had to click to select, everything was displayed in a huge list and now its accessibility is a problem. The best would be a combobox, a link that could collapse/expand or a button that could show the options.

- It is very common to have a "menu" button on almost every screen that, when pressed, is supposed to show several options. This is not accessible with screen readers. It can be made accessible with this same philosophy, but the current one does not behave properly.

Thanks for your attention and great work to improve homeassistant every day.

PS: please, I would strongly request to check the accessibility before uploading to production, as there are many users affected and it is a totally blocking issue for all of us.

### Describe the behavior you expected

Work fine with screenreaders and read all successfully!

### Steps to reproduce the issue

Use screenreader JAWS/NVDA in different browsers and check it.
2.
3.
...

### What version of Home Assistant Core has the issue?

Last versions

### In which browser are you experiencing the issue?

Chrome, firefox, edge for Windows with several screenreaders like JAWS and NVDA.

### Which operating system are you using to run this browser?

Windows 10 Pro

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### JavaScript errors shown in your browser console/inspector

```txt

```

**Comment by afr-ux** [comment]
Thank you very much for your reply.
Sorry. You are right. I see how the accessibility is getting worse in the homeassistant interface and I let myself get angry. It is better to go little by little, improving the interface progressively.

I have installed version 2025.6.3. I think that since 2025.6.0 is when the side menu changed. Now, with my screen reader, I can read:
- Toggle side panel (button) (Sorry if it is not exactly the same in English). I can press it, and it shows/hides a list with the different sections. But it doesn't tell me if it is expanded or collapsed. Better if it said "show side panel" / "hide side panel", as appropriate.
- In the list with the different sections appear items, but it does not inform that you can interact with them. They should be identified as a link to be able to click on them. In fact, if I press enter on them, nothing happens. Please note that I have my screen reader enabled.
Thank you very much for your support and understanding.
PS: Many of the components used in the interface I see are reused in many other screens, so by correcting it in one place, I am sure it will be corrected in many others.

**Comment by karwosts** [comment]
With attention to the sidebar, this is what I see with Chrome & NVDA:

I first load the page and I'm taken to my home screen. Ctrl+Home takes me to the toggle sidebar button, and NVDA reads:

> complementary landmark    clickable  button    Sidebar toggle

I press down, and it takes me to the adjacent Home Assistant logo, NVDA reads: 
> Home Assistant

I press down to get into the sidebar list, NVDA reads: 
> list  with 9 items  clickable  Main

Press down arrow, NVDA reads: 
> clickable  Energy

Press down arrow, NVDA reads:  
> clickable  Logbook

I press enter and my screen navigates and opens the logbook panel, NVDA reads:
> Logbook

Does that sound more like what you would expect to happen? I think it sounds better behaved than what you described, though I'm not sure the reason we saw different behavior.

**Comment by afr-ux** [comment]
This is, more or less, but here more information:
1- First element is: "complementary landmark clickable button Sidebar toggle".
But I don't know if it is pressed or not. I don't know if there is a list with my 18 items or not. I have to read more lines to know it.
Solution is, for example, use aria-pressed to share if the button is pressed or not.
2- In list with each item:
With NVDA I can read some empty lines between items. Not always. Between some items are empty lines, and it is not correct.
NVDA says "clickable" element, but JAWS doesn't say it. With JAWS you don't know that you can press in this item.
3- With NVDA you can press enter in an option, but you don't know if any has happen. Nothing happen. Correct is jump to header with information begin.
With JAWS you don't know if you can press or not, but if you press enter in any item, nathing happen, don't work fine.
I would like share a video, but I prefer any private way to send it.
Thank you very much.
